README.md

```diff
@@ -78,7 +78,7 @@ If you get stuck, don't worry we are here to help.
 The best order of communication is:
 
  * [neptune community spectrum](https://spectrum.chat/neptune-community)
- * [neptune-mlflow docs](https://docs.neptune.ai/integrations/mlflow.html)
+ * [neptune-mlflow docs](https://docs.neptune.ai/integrations/mlflow)
  * [github issues](https://github.com/neptune-ai/neptune-mlflow/issues)
 
```

Comment:
* [Email](mailto:support@neptune.ai)

Reply:
This PR just fixes the docs link. The page clearly needs a major overhaul